dask_sql/physical/rex/core/call.py

```diff
@@ -51,6 +51,10 @@ def as_timelike(op):
         raise ValueError(f"Don't know how to make {type(op)} timelike")
 
 
+def is_timestamp_nano(obj):
+    return "int" in str(type(obj)) or "int" in str(getattr(obj, "dtype", ""))
```

Comment:
Would it make sense to use something like `pd.api.types.is_integer_dtype` for this check, or is there a specific case I'm missing?